### ISSUE TITLE
Replace deprecated bower

### ISSUE THREAD

**Issue by r-brown** [Labs64/laravel-boilerplate#7]
Following message is shown during bower installation
```
sh-4.2# npm install -g bower
npm WARN deprecated bower@1.8.0: ..psst! While Bower is maintained,
we recommend Yarn and Webpack for *new* front-end projects!
Yarn's advantage is security and reliability,
and Webpack's is support for both CommonJS and AMD projects.
Currently there's no migration path but we hope you'll help us figure out one.
/usr/bin/bower -> /usr/lib/node_modules/bower/bin/bower
/usr/lib
`-- bower@1.8.0
```

**Comment by r-brown** [comment]
https://bower.io

> ...psst! While Bower is maintained, we recommend [yarn](https://yarnpkg.com/) and [webpack](https://webpack.js.org/) for new front-end projects!

**Comment by v-rudkovskiy** [comment]
[Fixed in Labs64/laravel-boilerplate@385ff12](https://github.com/Labs64/laravel-boilerplate/commit/385ff1225215f7515d891e684b2bc4a24e526646)
